assets/js/src/common/categories/categories_item.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
```

Comment:
This can be written as
```
type Props = Category & {
  onSelect: (name: string) => any,
  active?: boolean
}
```
This way, if we change the `Category` type, this will update automatically.

Reply:
Fixed.